configs/beaglebone_black_debian_emmc_config

```diff
@@ -1,8 +1,7 @@
 # Binaries generated with the following script:
-#
-#    https://github.com/drewmoseley/mender-convert-integration-scripts/blob/master/build-uboot-bbb.sh
-#
-BEAGLEBONE_BLACK_BINARIES="beaglebone-black-integration-debian-emmc-2018.07.004.tar"
+#    https://github.com/mendersoftware/mender-convert-integration-scripts/blob/master/build-uboot-bbb.sh
+
+BEAGLEBONE_BLACK_CONFIG="beaglebone-black-debian-emmc"
```

Comment:
Also should this one follow the config name? You are now using `-` instead of underscore (`_`) which is used in the actual configuration name?

Reply:
Makes sense. I will rework it here and in https://github.com/mendersoftware/mender-convert-integration-scripts/pull/3